lib/engine/game/g_system18/map_britain_customization.rb

```diff
@@ -0,0 +1,497 @@
+          hexes.find { |h| h.id == 'G8' }.tile.cities[0].place_token(phoenix, phoenix.tokens[1], free: true)
+
+          @britain_mines = BRITAIN_MINE_HEXES.map { |h| hex_by_id(h) }
+          @britain_corps_with_mines = {}
+        end
+
+        def map_britain_company_header(_company)
+          'CHARTER'
+        end
+
+        def map_britain_init_round
+          map_britain_new_parliament_round
+        end
+
+        def map_britain_new_parliament_round
+          @log << "-- Parliament Round #{@turn} -- "
+          GSystem18::Round::Parliament.new(self, [
+            GSystem18::Step::CharterAuction,
+          ])
+        end
+
+        def map_britain_next_round!
+          @round =
+            case @round
+            when Engine::Round::Stock
+              map_britain_stock_round_finished
+              @operating_rounds = @phase.operating_rounds
+              reorder_players
+              new_operating_round
+            when Round::Operating
+              if @round.round_num < @operating_rounds
+                or_round_finished
+                new_operating_round(@round.round_num + 1)
+              else
+                @turn += 1
+                or_round_finished
+                or_set_finished
+                map_britain_new_parliament_round
+              end
+            else # Parliament Round
+              init_round_finished
+              new_stock_round
+            end
+        end
+
+        # remove un-excersized charters from players
+        def map_britain_stock_round_finished
+          @players.each do |player|
+            player.companies.dup.each do |c|
+              @log << "Right to open #{c.sym} lapses for #{player.name}"
+              player.companies.delete(c)
+              c.owner = nil
+            end
+          end
+        end
+
+        def map_britain_can_par?(corporation, entity)
+          !corporation.ipoed && entity.companies.find { |c| c.sym == corporation.name }
+        end
+
+        def map_britain_after_par(corporation)
+          entity = corporation.owner
+
+          company = entity.companies.find { |c| c.sym == corporation.name }
+          raise GameError, 'Logic error, no matching company found' unless company
+
+          entity.companies.delete(company)
+          company.close!
+        end
+
+        # can the corporation reach an icon?
+        def map_britain_can_remove_icon?(entity)
+          return false unless entity.corporation?
+          return false if @britain_corps_with_mines[entity.name]
+
+          @britain_mines.any? { |m| @graph.reachable_hexes(entity).include?(m) }
+        end
+
+        def map_britain_icon_hexes(entity)
+          return [] unless entity.corporation?
+          return [] if @britain_corps_with_mines[entity.name]
+
+          @britain_mines.select { |m| @graph.reachable_hexes(entity).include?(m) }.map(&:id)
+        end
+
+        def map_britain_remove_icon(entity, hex_id)
```

Comment:
Might it be easier to implement these as hex assignments instead of icons? Or hex tokens?

Reply:
I didn't see much difference between this and hex tokens.